accounts/models.py

```diff
@@ -132,11 +142,15 @@ def synchronize_facebook_friends(self):
 
     @property
     def statistics_dict(self):
+        if self.reputation:
+            reputation = "%s%%" % formatted(self.reputation)
```

Comment:
This is Ajax response
8 kwi 2016 23:31 "Wojciech Zając" <notifications@github.com> napisał(a):

> In accounts/models.py
> <https://github.com/KlubJagiellonski/Politikon/pull/131#discussion_r59092289>
> :
>
> > @@ -132,11 +142,15 @@ def synchronize_facebook_friends(self):
> >
> >      @property
> >      def statistics_dict(self):
> > +        if self.reputation:
> > +            reputation = "%s%%" % formatted(self.reputation)
>
> Why not {{ object.reputation|formatted }}% in template?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/KlubJagiellonski/Politikon/pull/131/files/d40fa5b58d95432d7b53878c4e997987225e563c#r59092289>
>